CMakeLists.txt

```
@@ -283,6 +283,14 @@ if(${CMK_USE_ZLIB})
   set(CMK_SYSLIBS "${CMK_SYSLIBS} -lz")
 endif()
 
+if(${CMK_USE_SHMEM})
+  if(${CMK_HAS_XPMEM})
+    set(CMK_SYSLIBS "${CMK_SYSLIBS} -lxpmem")
```

```suggestion
    set(CMK_SYSLIBS "$CMK_SYSLIBS -lxpmem")
```
See https://github.com/UIUC-PPL/charm/pull/3538